.circleci/config.yml

```diff
@@ -40,7 +40,11 @@ jobs:
       - run:
           name: Unsupervised Transcriptomic Analysis - Generate Plot Lists         
           command: ./scripts/run_in_ci.sh Rscript --vanilla analyses/transcriptomic-dimension-reduction/scripts/get-plot-list.R  --input_directory analyses/transcriptomic-dimension-reduction/results --filename_lead rsem_all --output_directory analyses/transcriptomic-dimension-reduction/plots --color_variable RNA_library
-    
+          
+      - run:
+          name: Unsupervised Transcriptomic Analysis - Multipanel Plots      
+          command: ./scripts/run_in_ci.sh bash analyses/transcriptomic-dimension-reduction/04-multipanel-plots.sh
```

Comment:
This shell script can be run in CI for two main reasons: 1) the files it acts on are committed to the repository so when the code gets checked out they will be there 2) even if that was not the case it would run over any files in that directory.

Reply:
Ahh okay, gotcha. So in that case, would we want to keep the R scripts for the first 3 executable parts of this directory in the final collective shell script or would we use each of the individual shell scripts?

Reply:
I would expect the shell script that gets implemented specifically for running this in CI would execute the following:

* A subset by strategy step
* The dimension reduction steps, but t-SNE should be skipped using the `--skip_tsne` flag for the polyA subset files because there probably aren't enough of them and perplexity needs to be lowered.
* Generating plot lists
* The multipanel plot list

It's the second step that diverges from how you would 'normally' run the pipeline for analysis.